### ISSUE TITLE
Run app on another computer

### ISSUE THREAD

**Issue by fauwzk** [austinbaccus/forza-telemetry#53]
Is it possible to run the app on a client that is different than the computer running FH5 ?

**Comment by austinbaccus** [comment]
@fauwzk yes, it is. In fact this is the main method used by most other Forza telemetry apps on GitHub.

I would love to add support for this feature, but I currently only have my desktop with me right now, so getting it to work would be tricky.

The code that handles this is in /ForzaCore/Program.cs, and you can configure the host and out ports on lines 21 and 22.

**Comment by Rugtert** [comment]
Change IPAddress.Loopback to IPAddress.Any on line 29 of /ForzaCora/Program.cs and it will listen to remote connections.
